messaging/content-templates/create-wa-otp-button-template/output/create-wa-otp-button-template.json

```diff
@@ -0,0 +1,27 @@
+{
+    "account_sid": "$TWILIO_ACCOUNT_SID",
+    "date_created": "2023-06-02T14:34:25Z",
+    "date_updated": "2023-06-02T14:34:25Z",
+    "friendly_name": "WhatsApp OTP button template",
+    "language": "en",
+    "links": {
+        "approval_create": "https://content.twilio.com/v1/Content/HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/ApprovalRequests/whatsapp",
+        "approval_fetch": "https://content.twilio.com/v1/Content/HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX/ApprovalRequests"
+    },
+    "sid": "HXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX",
+    "types": {
+        "whatsapp/authentication": {
+            "actions": [
+                {
+                    "copy_code_text": "Copy verification code",
+                    "type": "COPY_CODE"
+                }
+            ],
+            "add_security_recommendation": true,
+            "body": "{{1}}",
+            "code_expiration_minutes": 40
```

Comment:
This expires in 40, but in the curl, it's 30?

Reply:
Good catch!